### PR TITLE
refactor(multiple): switch to input transforms

### PR DESCRIPTION
**Pull request by mhaertwig**
Reworks components to use input transforms and removes unused mixins.

### CODE REVIEW

**Review by kyubisation — APPROVED** [review]
LGTM 👍 
Nice improvement! 😃 
Should we mark `mixinTabIndex` (etc.) as deprecated?

**Comment by mhaertwig** [comment]
> LGTM 👍 Nice improvement! 😃 Should we mark `mixinTabIndex` (etc.) as deprecated?
Good idea. I added an annotation.

**Comment by mhaertwig** [comment]
I've removed the remaining `mixinTabindex` and `mixinDisabled` mixins and marked both as deprecated 👍

File: src/angular/icon/icon.ts [hunk 1]
```diff
@@ -126,13 +125,7 @@ export class SbbIcon implements OnInit, AfterViewChecked, OnDestroy {
...
    */
   @Input()
-  get inline(): boolean {
-    return this._inline;
-  }
-  set inline(inline: BooleanInput) {
-    this._inline = coerceBooleanProperty(inline);
-  }
-  private _inline: boolean = false;
+  inline: boolean = false;
```
[the remark this answers is not in the document]
Comment (mhaertwig) [reply src/angular/icon/icon.ts:129]: How?! 😆

[the remark this answers is not in the document]
Comment (jeripeierSBB) [reply src/angular/icon/icon.ts:129]: it's a boolean prop?

**Pull request merged**